src/screens/BodyAnalysisScreen.tsx

```diff
@@ -1,59 +1,234 @@
+                <Text
+                  style={{
+                    ...styles.subtitleUpperCase,
+                    color: theme.colors.onBackground,
+                    marginLeft: 20,
+                    marginBottom: 15,
+                    marginTop: 20,
+                  }}
+                >
+                  Target
+                </Text>
+                <StackRow>
+                  <Text
+                    style={{
+                      ...styles.secondarySubtitleUppercase,
+                      color: theme.colors.onBackground,
+                      alignSelf: "center",
+                      marginRight: 10,
+                    }}
+                  >
+                    Body Fat:
+                  </Text>
+                  <Text
+                    style={{
+                      ...styles.bigUnits,
+                      color: theme.colors.onBackground,
+                    }}
+                  >
+                    {geminiResponse?.target.bodyFat}%
+                  </Text>
+                </StackRow>
+                <StackRow>
+                  <Text
+                    style={{
+                      ...styles.secondarySubtitleUppercase,
+                      color: theme.colors.onBackground,
+                      alignSelf: "center",
+                      marginRight: "auto",
+                    }}
+                  >
+                    Weight:
+                  </Text>
+                  <Text
+                    style={{
+                      ...styles.bigUnits,
+                      color: theme.colors.onBackground,
+                    }}
+                  >
+                    {geminiResponse?.target.weight}
+                  </Text>
+                  <Text
+                    style={{
+                      ...styles.secondarySubtitleUppercase,
+                      color: theme.colors.onBackground,
+                      marginLeft: 4,
+                    }}
+                  >
+                    kg
+                  </Text>
+                </StackRow>
+              </View>
+            </StackRow>
+            <Surface
+              style={{ ...styles.surface, width: "70%", marginVertical: 20 }}
+              elevation={4}
+            >
+              <Text
+                style={{
+                  ...styles.textBackground,
+                  color: theme.colors.onBackground,
+                }}
+              >
+                {geminiResponse?.additionalInfo}
+              </Text>
+            </Surface>
+          </View>
+        )}
+        <StackRow>
+          <Button
+            mode="contained"
+            onPress={() => navigation.goBack()}
+            style={{
+              marginTop: 20,
+              marginBottom: 20,
+              marginRight: 20,
+              marginLeft: "auto",
+            }}
+          >
+            Complete
+          </Button>
+          <Button
+            mode="contained"
+            onPress={fetchAnalysis}
```

Comment:
So what shows up if the user navigates to this screen and they don't press the button? 

If the user has had no analysis done... we should automatically trigger the analysis to be done (on page load). If the user has previous had an analysis done... this information should be displayed and yes, they can press this button to re-analysis (ideally only if they've made changes to their data).